### ISSUE TITLE
GSoCer's blog IIIT Bh

### ISSUE THREAD

**Issue by Rupeshiya** [p-society/goals#29]
I think there should also be a blog which  contains  the journey from start to end ,experience and suggestions for GSoC aspirants  and for any other info we can contact GSoCer's of our college too.
What do you think @palash25  ?

**Comment by palash25** [comment]
Nice idea but every GSoCer writes their own blog, so writing or copy pasting articles for another blog leads to duplication of effort which I don't like.

I would like to suggest a few modifications to your idea. Instead of building a blog from scratch you might want to consider building a blog aggregator site that collects and archives blog from every GSoCer's RSS feed. 

This seems more practical and doable and also automates the boring stuff. :wink: 

Thanks for your contribution :smile:  @Rupeshiya

**Comment by Rupeshiya** [comment]
yep it will be better to have that.👌👍 .

**Comment by Rupeshiya** [comment]
Definately we hope too.

**Comment by gupta2140** [comment]
Late to comment, but you can build a crawler and add the URLs of the RSS feeds to a file. Then set it whatever duration you want and it will fetch all those GSoC blogs per cycle and list them/update up on your aggregator whenever a new post is needed. You can have a look at planet.kde.org or dot.kde.org  These are perfect and awesome working example of what you are looking for..

**Comment by palash25** [comment]
That is exactly what I just said refer: https://github.com/p-society/goals/issues/29#issuecomment-386791777

**Comment by gupta2140** [comment]
Yep, I just gave some sources which he can take inspiration from and understand its proper working.
